src/callbacks/onLayout/initStudyDayControl.js

```diff
@@ -14,7 +14,7 @@ export default function initStudyDayControl() {
 
     //set min and max values and add annotations
     chart.controls.studyDayRange = d3.extent(
-        chart.imputed_data.filter(d => d.analysisFlag),
+        chart.imputed_data.filter(f => f.key_measure),
```

Comment:
use all study days on which measurements of the key medical signs were taken right